contrib/systemd/origin-master.service

```diff
@@ -19,4 +18,3 @@ Restart=always
 
 [Install]
 WantedBy=multi-user.target
-WantedBy=origin-node.service
```

Comment:
I think we went with WantedBy because currently a master is always a node but a node isn't always a master. So it was emitting a warning if we placed a Wants=origin-master.service in origin-node.service. I'll try to find the PR where we hashed this all out originally.